### PR TITLE
feat: auto estimate fee for station

### PR DESCRIPTION
**Pull request by llllllluc**

### CODE REVIEW

File: src/providers/TerraStationProvider.ts [hunk 1]
```diff
@@ -307,10 +307,30 @@ export const TerraStationProvider = class TerraStationProvider implements Wallet
...
+        const stdFee = calculateFee(
+          Math.round(gasEstimation * DEFAULT_GAS_MULTIPLIER),
+          network.gasPrice || DEFAULT_GAS_PRICE,
+        );
+        fee = JSON.stringify({
+          fee: stdFee.amount[0],
+          gasLimit: stdFee.gas,
+        });
+      } catch (error: any) {
+        throw error;
```
Comment (llllllluc) [inline src/providers/TerraStationProvider.ts:326]: not sure how we want to handle the error here

**Comment by afsardo** [comment]
Feature added in the following commit https://github.com/delphi-labs/shuttle/commit/fc5d313dea7284a0dab7f19f91b1847c96c9137a.

**Comment by afsardo** [comment]
Merged in release [v.2.16.0](https://github.com/delphi-labs/shuttle/releases/tag/v.2.16.0)

**Comment by llllllluc** [comment]
thank you!!!

**Pull request closed without merging**